wovnrb.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   #spec.extensions    = spec.files.grep(%r{/extconf\.rb$})
 
   spec.add_dependency "nokogumbo", ">= 1.3.0"
-  spec.add_dependency "nokogiri", "<= 1.8"
+  spec.add_dependency "nokogiri", "<= 1.8.1"
```

Comment:
Is it still under the range of security issue?
![image](https://user-images.githubusercontent.com/2409560/35499405-84c31526-0515-11e8-91bd-49c6635ba7b8.png)

Reply:
As I understand "update suggested" means it will fix the vulnerability found, what do you understand?

Reply:
Yes I understand so, too.
So it means we should update the dependency to `~> 1.8.1`

Reply:
ahhh got it, I will do it thanks lol